src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/commands/mobile/on_vehicle_data_notification.cc

```diff
@@ -89,6 +89,11 @@ void OnVehicleDataNotification::Run() {
         SDL_LOG_ERROR("NULL pointer");
         continue;
       }
+
+      if (name == strings::tire_pressure) {
```

Comment:
@ychernysheva please swap the values here

Reply:
Done in f40c97b